.ci/azure/linux.yml

```diff
@@ -226,6 +226,7 @@ jobs:
       cmakeArgs: >
         -GNinja
         -DCMAKE_VERBOSE_MAKEFILE=ON
+        -DENABLE_OV_PADDLE_FRONTEND=OFF
```

Comment:
I suppose we still need to build it.
requirements.txt files affect only tests execution. No converted models => tests will fail, but we are still able to build tests and FE itself.

Reply:
Change reverted.